setup.cfg

```diff
@@ -50,6 +50,17 @@ dev =
     ruff
     check-manifest
     %(testing)s
+doc =
+    sphinx>6
+    sphinx-autobuild
+    sphinx-external-toc
+    sphinx-copybutton
+    sphinx-gallery
+    sphinx-favicon
+    sphinxcontrib-video
```

Comment:
FYI I am using the sphinxcontrib-video plugin for convenience, although we don't have to if we prefer not. https://github.com/sphinx-contrib/video